### DISCUSSION TITLE
No timescaledev/timescaledb-toolkit:nightly found

### DISCUSSION THREAD

**Discussion by Maartendohmen** [timescale/timescaledb-toolkit#231] [Problem Discussions]
Hi all, I have a question concerning the timescaledb-toolkit. Looking at the wiki, it states the following:
_We also provide nightly builds as a docker images in timescaledev/timescaledb-toolkit:nightly._

I can't find however no images on dockerhub or anywhere else. Looking at the last few nightly's, they all are failing. Were can I find this image? And if it's unavailable for now, is there an ETA when I might become available again?

Thanks

**Comment by JLockerman** [comment]
That’s strange, it’s [being built](https://github.com/timescale/timescaledb-toolkit/runs/3419617946?check_suite_focus=true) but it isn’t making it to dockerhub for some reason…
I’ll have to look more into it, I’ll try to have it back up Friday or Monday

**Comment by JLockerman** [comment]
Update: if PR https://github.com/timescale/timescaledb-toolkit/pull/232 works it’ll be up when sooner, we’ll know in 20 minutes or so.

**Comment by JLockerman** [comment]
Image is up at `timescaledev/timescale-analytics:nightly`. It turns out the bot doesn’t have permission for the timescaledb-toolkit repo so the image will live in the analytics tag until I can figure out why.

**Comment by JLockerman** [comment]
(Readme will be updated with thus when the PR merges)

**Comment by Maartendohmen** [comment]
Okay clear, thanks for the info

**Comment by Maartendohmen** [comment]
@JLockerman Are there any plans of adding the timescaledev/timescale-analytics on dockerhub with release versions? We would like to use stable versions instead of nightly's if that's possible.

**Reply by jerryxwu** [reply]
@Maartendohmen Currently, the best option for a stable version of the toolkit docker is [timescaledb-ha](https://hub.docker.com/r/timescale/timescaledb-ha). We have this documented in the README. But it's really easy to miss it.
